### PR TITLE
Protocol templates - revert action and alert message [SCI-7910]

### PR DESCRIPTION
**Pull request by sboursen-scinote**
Jira ticket: [SCI-7910](https://scinote.atlassian.net/browse/SCI-7910)

### What was done
Updated the text for the revert modal.

### Note:
This contribution/PR depends on #4975. It would be better if that was merged first.

[SCI-7910]: https://scinote.atlassian.net/browse/SCI-7910?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/assets/javascripts/my_modules/protocols.js [hunk 1]
```diff
@@ -99,12 +99,23 @@ function initLinkUpdate() {
   var updateBtn = modal.find(".modal-footer [data-action='submit']");
   $("[data-action='unlink'], [data-action='revert'], [data-action='update-parent'], [data-action='update-self']")
     .on('ajax:success', function(e, data) {
-      modalTitle.html(data.title);
-      modalBody.html(data.message);
-      updateBtn.text(data.btn_text);
-      modal.attr('data-url', data.url);
+      const modalBodyTemplate = (message, warning) => `<p class="message">${message}</p>
```
Comment (aignatov-bio) [inline app/assets/javascripts/my_modules/protocols.js:102]: Just combine message and warning in one I18n record

File: app/assets/stylesheets/my_modules/protocols/index.scss [hunk 2]
```diff
@@ -680,3 +682,52 @@
   display: inline-block;
   margin-left: 4px;
 }
+
+.confirm-link-update-modal {
```
Comment (aignatov-bio) [inline app/assets/stylesheets/my_modules/protocols/index.scss:686]: Do not invent a new modal, just use the current. There is just an error In mockup.

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/my_modules/protocols.js (hunk 1 above)
Comment (sboursen-scinote) [reply app/assets/javascripts/my_modules/protocols.js:102]: Message and Warning have different styles.

File: app/assets/stylesheets/my_modules/protocols/index.scss (hunk 2 above)
Comment (sboursen-scinote) [reply app/assets/stylesheets/my_modules/protocols/index.scss:686]: The new Figma style is different for protocol repository revamp

File: app/assets/javascripts/my_modules/protocols.js [hunk 3]
```diff
@@ -99,12 +99,20 @@ function initLinkUpdate() {
   var updateBtn = modal.find(".modal-footer [data-action='submit']");
   $("[data-action='unlink'], [data-action='revert'], [data-action='update-parent'], [data-action='update-self']")
     .on('ajax:success', function(e, data) {
-      modalTitle.html(data.title);
-      modalBody.html(data.message);
-      updateBtn.text(data.btn_text);
-      modal.attr('data-url', data.url);
+      const {
```
Comment (aignatov-bio) [inline app/assets/javascripts/my_modules/protocols.js:102]: You can just revert as it was

File: app/controllers/protocols_controller.rb [hunk 4]
```diff
@@ -881,6 +881,7 @@ def unlink_modal
         render json: {
           title: t('my_modules.protocols.confirm_link_update_modal.unlink_title'),
           message: t('my_modules.protocols.confirm_link_update_modal.unlink_message'),
+          warning: t('my_modules.protocols.confirm_link_update_modal.unlink_warning'),
```
Comment (aignatov-bio) [inline app/controllers/protocols_controller.rb:884]: What it is?

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

File: config/locales/en.yml [hunk 5]
```diff
@@ -1074,18 +1074,18 @@ en:
           update_self_title: "Update this protocol with the version stored in the templates"
       confirm_link_update_modal:
         unlink_title: "Unlink protocol template"
-        unlink_message: "Are you sure you want to unlink the task protocol from the templates version? This will stop any tracking of changes."
-        unlink_btn_text: "Unlink"
-        revert_title: "Revert protocol"
-        revert_message: "Are you sure you want to revert the task protocol to the templates version? This will override any local changes you made."
-        revert_btn_text: "Revert"
+        unlink_message: "This will stop tracking changes of protocol templates. We can’t link it back once you unlink it."
```
Comment (aignatov-bio) [inline config/locales/en.yml:1077]: https://github.com/scinote-eln/scinote-web/pull/4975/files

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

**Review by aignatov-bio — APPROVED** [review]

**Pull request merged**